workspace/resource_notebook.go

```diff
@@ -291,16 +303,32 @@ func ResourceNotebook() common.Resource {
 					if err != nil {
 						return err
 					}
-					err = notebooksAPI.Create(createNotebook)
+					resp, err = notebooksAPI.Create(createNotebook)
 				}
 				if err != nil {
 					return err
 				}
 			}
+			if d.Get("object_type").(string) == "" {
+				d.Set("object_type", Notebook)
```

Comment:
Why do we need to know the object type here? Isn't it always notebook for a databricks_notebook? Or is it because this can be a DBC file which can contain other stuff as well?

I am also missing when it would be set before (when this `if` block would be skipped). It's marked as `deprecated` and should always be notebook, I guess?